apps/site/next-data/generators/supportersData.mjs

```diff
@@ -0,0 +1,95 @@
+// This is used to ensure that URLs are always in the correct format
+function fixUrl(url) {
```

Comment:
Doesn't seem like this should be here, like in this file. Also, why we need to replace these to https? The change to https should be done automatically by the website/url if they support https, otherwise we're forcefully switching to https schema when a website could maybe not even support it.

Reply:
cc @canerakdas